tests.js

```diff
@@ -0,0 +1,88 @@
+require('required_env')([ 'AUTH_URL' ]);
+import Driver from './src';
+import test from 'blue-tape';
+import bandname from 'bandname';
+
+// Set to your auth api key
+const TEST_PASSWORD = 'hai';
```

Comment:
I pulled it out of the URL because the auth service can use basic auth flow or JWT tokens, and I thought `auth-driver` should be extendable enough to handle both, maybe not now, but in the future.

An alternative here is to allow the url to have basic auth within it, the reasoning I didn't go with this was to be a bit more explicit.

Eg: 

```
const driver = new Driver('https://authserver.com', {
  username: 'bob',
  password: 'jane'
})
```

```
const driver = new Driver('https://authserver.com', {
  jwtSecret: 'whatever'
})
```

would both work without breaking the api.